in_toto/gpg/common.py

```diff
@@ -329,14 +672,18 @@ def parse_signature_packet(data):
     if subpacket_type == PARTIAL_KEYID_SUBPACKET:
       short_keyid = binascii.hexlify(subpacket_data).decode("ascii")
 
+    # info["subpackets"].append((
+    #   subpacket_type,
+    #   binascii.hexlify(subpacket_data).decode("ascii")))
```

Comment:
This somehow sneaked through. Could you please remove it?

Reply:
I left it for michael, but I can remove it.

Reply:
I added a TODO comment 60555c7 and will remove or un-comment it with #245